### PR TITLE
Add full system config

### LINKED ISSUE #8: Write Terraform configuration for the (current) full system
The (current) full system includes deploying:
- `heimdall`
- `karen`
- `ether`
- `patches`
- `me-you`

### PR DESCRIPTION
**Pull request by igorveselinovic**
Resolves #8 

It's very similar to the `patches` integration environment. The major difference is that this also deploys `me-you` in its own separate ECS cluster.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -48,3 +181,129 @@ module "timescaledb_instance" {
...
+  db_username     = var.rds_username
+  db_password     = var.rds_password
+  kafka_server    = split(",", aws_msk_cluster.main.bootstrap_brokers)[0]
+  kafka_topic     = "updates"
+  karen_endpoint  = module.karen.elb_dns_name
+  efs_id          = aws_efs_file_system.ether.id
+}
+
+module "patches" {
+  source            = "github.com/schramm-famm/patches?ref=sprint05//terraform/modules/patches"
```
Comment (igorveselinovic) [inline main.tf]: Will update this once `sprint05` is merged into `master` in `patches`.

File: main.tf [hunk 2]
```diff
@@ -48,3 +181,129 @@ module "timescaledb_instance" {
...
+  kafka_server      = split(",", aws_msk_cluster.main.bootstrap_brokers)[0]
+  kafka_topic       = "updates"
+  heimdall_endpoint = module.heimdall.internal_lb_dns_name
+  ether_endpoint    = module.ether.elb_dns_name
+}
+
+/* FRONTEND SERVICE CONFIG */
+
+module "me_you" {
+  source          = "github.com/schramm-famm/me-you?ref=sprint05//terraform"
```
Comment (igorveselinovic) [inline main.tf]: Will update this once `sprint05` is merged into `master` in `me-you`.

File: main.tf [hunk 3]
```diff
@@ -4,29 +4,162 @@ provider "aws" {
...
+  }
+}
+
+resource "aws_security_group" "efs" {
+  name        = "${var.name}_efs"
+  description = "Allow NFS traffic into EFS mount targets"
+  vpc_id      = module.ecs_base.vpc_id
+
+  ingress {
+    from_port       = 2049
```
Comment (Thao-Tran) [inline main.tf:150]: Do androids dream of electric sheep though?

**Review by Thao-Tran — APPROVED** [review]

File: main.tf (hunk 3 above)
Comment (igorveselinovic) [reply main.tf:150]: ⚡🐑

**Review by Thao-Tran — APPROVED** [review]

**Review by Thao-Tran — APPROVED** [review]

**Pull request merged**